CONTRIBUTING.md

```diff
@@ -159,10 +158,6 @@ Once you've updated the test suite, there are a few things you'll want to check.
 You can also take a look at the `.travis.yml` file to see what the continuous
 integration system will do to verify the track.
 
-Take a look at our [pull request guidelines](#pull-request-guidelines).
-You don't need to get it perfect the first time around; we'll work with you to
-get the patch merged.
-
```

Comment:
> […] documentation of how to write a pull request has been extracted from CONTRIBUTING.md and added to a separate PULL_REQUEST_TEMPLATE.md document?

@Insti: I have extracted barely more than links (above & below this comment). The actual advice is still at the link targets. How about not deleting those lines here, but duplicating them in the PR template?